libs/domain/feature-gateway/src/operation/CreateAPIKey.ts

```diff
@@ -0,0 +1,70 @@
+import { UnknownError } from "@banklessdao/util-data";
```

Comment:
The `ContentGateway` interface was split into individual `Operation`s like this one. Now `ContentGateway` acts as a [Facade](https://en.wikipedia.org/wiki/Facade_pattern#:~:text=The%20facade%20pattern%20(also%20spelled,complex%20underlying%20or%20structural%20code.).